src/RadworksGovernor.sol

```diff
@@ -174,11 +178,17 @@ contract RadworksGovernor is
     return GovernorTimelockCompound.state(proposalId);
   }
 
-  /// @notice The amount of RAD required to meet the quorum threshold for a proposal
-  /// as of a given block.
+  /// @notice The amount of RAD required to meet the quorum threshold for a proposal.
   /// @dev Our implementation ignores the block number parameter and returns a constant.
-  function quorum(uint256) public pure override returns (uint256) {
-    return QUORUM;
+  function quorum(uint256) public view override returns (uint256) {
+    return _quorum;
+  }
+
+  /// @notice Set the amount of RAD required to meet the quorum threshold for a proposal.
+  /// @dev This function is only callable by executed governance proposals (thus, the Timelock).
+  function setQuorum(uint256 _newQuorum) public onlyGovernance {
```

Comment:
It would also be good to have a test that  doesn't use a proposal to call this method.

Reply:
Done: bdc426a8558ec5571abb46949b072e5dfdbfb81d